docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Sphinx>=2.1.2
+git+https://github.com/rwblair/sphinxcontrib-versioning.git@39b40b0b84bf872fc398feff05344051bbce0f63#egg=sphinx-versioning
+sphinxcontrib-apidoc
+# workflows autodocs were failing without pytest
+pytest
```

Comment:
maybe exclude all test files and the `conftest.py` file? If that doesn't work, then add `pytest` to the mocked imports?

Reply:
Removed it from the requirements and the build succeeded.